commands/visionCommand.py

```diff
@@ -0,0 +1,65 @@
+import wpilib
+import commands2
+import commands2.cmd
+import wpimath.controller
+
+from subsystems.photonVisionSubsystem import visionSub
+
+import constants
+
+import logging
+logger = logging.getLogger("Jhony")
+        
+class takeSnapShot(commands2.Command):
+    def __init__(self, visionSubsystem: visionSub) -> None:
+        super().__init__()
+        self.vision_sub = visionSubsystem
+
+    def initialize(self):
+        logger.info("Capturing Image")
+
+    def execute(self):
+        self.vision_sub.captureImage()
+
+    def isFinished(self):
+        # command does not finish it needs to be cancelled
+        return False
+
+    def end(self, interrupted: bool):
+        self.vision_sub.nothingCommand()
+        
+class togglePipeline(commands2.Command):
+    def __init__(self, visionSubsystem: visionSub) -> None:
+        super().__init__()
+        self.vision_sub = visionSubsystem
+
+    def initialize(self):
+        logger.info("Changing Pipeline")
+
+    def execute(self):
+        self.vision_sub.togglePipeLine()
+
+    def isFinished(self):
+        # command does not finish it needs to be cancelled
+        return False
+
+    def end(self, interrupted: bool):
+        self.vision_sub.nothingCommand()
+        
+class doNothing(commands2.Command):
+    def __init__(self, visionSubsystem: visionSub) -> None:
+        super().__init__()
+        self.vision_sub = visionSubsystem
+
+    def initialize(self):
+        logger.info("Changing Pipeline")
+
+    def execute(self):
+        self.vision_sub.nothingCommand
```

Comment:
Here we're not intending to talk about `nothingCommand`, we're intending to invoke it.  When you invoke a method or function, you need parentheses.  If the method/function takes parameters, those parameters go inside the parentheses.  But even if the method/function doesn't take any parameters, you still need the parentheses.